.github/workflows/adhoc_fuzzer.yml

```diff
@@ -0,0 +1,245 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "Adhoc Fuzzer Jobs"
+
+on:
```

Comment:
It would be good to also add `pull_request` with a path filter set to this workflow that changes to the workflow can be tested when submitting a PR but that will also require some additional defaults as there will be no inputs iirc.